app/views/shared/_info_content.html.erb

```diff
@@ -35,11 +35,24 @@
   <% if @campaign.members.empty? %>
     <p>Pas encore de membre. Soyez le premier 😊.</p>
   <% end %>
-  <% @campaign.members.uniq.each do |member| %> <%# uniq to avoid showing several times the same pic in case a member has several missions for this campaign %>
-    <% if member.photo.attached? %>
-      <%= cl_image_tag member.photo.key, height: 40, width: 40, crop: :fill, class: "avatar mx-2"%>
-    <% else %>
-      <img src="https://images.unsplash.com/photo-1536939459926-301728717817?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80" alt="" class= "avatar mx-2">
+  <% if @campaign.members.uniq.count > 6 %>
+    <% @campaign.members.uniq.first(5).each do |member| %> <%# uniq to avoid showing several times the same pic in case a member has several missions for this campaign %>
```

Comment:
cette query devrait être, à minima, dans le controller: `@first_5_members = @campaign.members.uniq.first(5)`.

Reply:
Won't do because I have to do it in 2 controller instead of 1 view.